chia/full_node/mempool_manager.py

```diff
@@ -156,6 +156,33 @@ def remove_seen(self, bundle_hash: bytes32):
         if bundle_hash in self.seen_bundle_hashes:
             self.seen_bundle_hashes.pop(bundle_hash)
 
+    def get_min_fee_increase(self):
+        # 0.00001 XCH
```

Comment:
would prefer this to be part of the config, but it's easy to change later. (not part of consensus)